files/fr/glossary/code_splitting/index.html

```diff
@@ -0,0 +1,26 @@
+---
+title: Code splitting
+slug: Glossary/Code_splitting
+tags:
+  - Glossary
+  - Reference
+  - Web Performance
+  - code splitting
+  - latency
+translation_of: Glossary/Code_splitting
+original_slug: Glossaire/Code_splitting
```

Comment:
original_slug is useful only if you changed the slug for the translated page, so you can safely remove this line :)